docs/transcribe.md

```diff
@@ -6,58 +6,57 @@ authors:
 
-<!-- todo: move "obtaining a model" to predict.md -->
-### Obtaining a model
+Transcription is a step that comes after [manually](segment.md) or [automatically segmenting](predict.md#predict-the-segmentation) the image. Make sure to review these pages first if necessary.
 
-You need a model to transcribe automatically, here are some ways to get one:  
+Manual transcription is usually necessary to produce training materials or when reviewing the result of an automatic transcription. The following section introduces the two panels dedicated to creating or modifying the transcription, as well as some related features.
 
-- by [training one with eScriptorium](walkthrough_train.md) or (via the command line) with [kraken](https://kraken.re),
-- by having one shared with you, either within [eScriptorium](walkthrough_collaborate.md#share-a-model), or sent to you as a file through other means of communication,
-- by importing a model from a file. Such files can be retrieved on the [Zenodo community](https://zenodo.org/communities/ocr_models/) dedicated to Kraken models.
+## Where to find the transcription features
 
-### Predicting the transcription
+To work on the transcription manually, you need to be on the "Edit" tab of a document dashboard (`{base_url}/document/{document-id}/images/`), which is available:
 
-To learn how to apply the text recognition model and obtain a prediction, see ["Predict a segmentation or a transcription"](walkthrough_predict.md).  
+- by clicking on the "Edit" button once you are in your document dashboard (it will send you to the last edited page of the document),
+- or by clicking on the "Edit" button of a page thumbnail, in the Images tab
 
-## Manual transcription
+Two panels allow you to manually modify the transcription:
 
-To work on the transcription manually, you need to be on the "Edit" tab of your document, which is available either by clicking on the "Edit" button once you are in your document or in a square when you hover upon the thumbnail of an image once your are in the following URL: `{base_url}/document/{document-id}/images/`.
+- the 4th panel, called "Transcription" (++control+4++),
+- and the 5th panel, called "Text" (++control+5++).
 
-Out of the five panels available in eScriptorium, two allow you to create and edit the transcription: "Transcription" and "Text".
+![image: Screenshot of the 5 activated panels (Transcription panel framed in red, Text panel framed in blue) ](img/transcribe/transcription_text_panels.png "The 4th and the 5th panels are for manually editing the transcription")
 
 !!! Tip
-    If you need to have special characters in your transcription and they are not available on your own keyboard, you can set and use a [virtual keyboard](walkthrough_virtual_keyboard.md). 
+    If your requires special characters, you can set and use a [virtual keyboard](virtual_keyboard.md).
```

Comment:
There is either an extra "r" at "you" or you are missing a word such as "transcription" or "text"

Reply:
I was missing a word in deed!